README.md

```diff
@@ -421,6 +422,25 @@ Notice: The following `data-uri` attributes
 ```
 would trigger two calls for data because of the difference in query strings, even though the path to service is the same in both.
 
+## Service response status code
+Snippet author is able to see service response status code.
```

Comment:
Should be rather something like: "Service response status code can be used in snippets e.g. as condition to display messages if response is not success".

See below other comment related to the above.